gen3-integration-tests/pages/study_viewer.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import os
+from utils import logger
+import utils.gen3_admin_tasks as gat
+
+from utils.test_execution import screenshot
+from playwright.sync_api import Page, expect
+
+get_index = gat.get_study_viewer_index(pytest.namespace)
```

Comment:
can't we use `get_portal_config` (https://github.com/uc-cdis/gen3-code-vigil/blob/master/gen3-integration-tests/utils/gen3_admin_tasks.py#L13) and filter this from the json